query/graphql/src/main/java/org/codice/ddf/admin/graphql/transform/GraphQLTransformScalar.java

```diff
@@ -17,14 +17,32 @@
 import org.codice.ddf.admin.graphql.GraphQLTypesProviderImpl;
 
 import graphql.Scalars;
+import graphql.schema.Coercing;
 import graphql.schema.GraphQLScalarType;
 
 public class GraphQLTransformScalar {
 
     GraphQLTypesProviderImpl<GraphQLScalarType> scalarTypesProvider;
+    private Coercing encryptPasswordCoercing;
```

Comment:
can this be private static final or at least final? I don't think this will be changing.